### PR TITLE
Create Django app

### LINKED ISSUE #18: Merge 'static' and 'Resources' folders
Merge 'static' and 'Resources' folders
Update paths in relevant files (like the logo image, etc...)

### PR DESCRIPTION
**Pull request by orzionpour**
[-] Create django app named 'falconapp'.
[-] Add the above app to the project configurations.
[-] Add falcon's login page to the above app.
[-] Add another static directory to configurations.

Resolve #18, #19

### CODE REVIEW

**Comment by orzionpour** [comment]
@kobihk @natifridman @nunnatsa @mgold1234 @avraham1992 @AvivLiberman @aradz1997 
Can you CR please?

This PR should fix Barak's comment from slack about the missing django app.
Note that the flake8 tests are failing due to django's auto-generated files which imports few modules but there is no use in them yet.

**Comment by nunnatsa** [comment]
Please fix flake8 new warnings

**Comment by orzionpour** [comment]
The flake8 tests are failing due to django's auto-generated files which imports few modules but there is no use in them yet since they are related to the Model which isn't implemented yet. Delete these lines?

**Comment by nunnatsa** [comment]
> The flake8 tests are failing due to django's auto-generated files which imports few modules but there is no use in them yet since they are related to the Model which isn't implemented yet. Delete these lines?

Usually we don't want to touch auto generated code. Mostly because it may cause unexpected issues, also, it may happen and again. Not sure about django and what it will do, though. Maybe it's possible to add some mock-temporary code to use these imports, if removing these lines won't work.

File: falconapp/views.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from django.shortcuts import render
 
+
 def login(request):
-    return render(request, 'login.html')
\ No newline at end of file
+    return render(request, "falconapp/login.html")
```
Comment (mgold1234) [inline falconapp/views.py:5]: under falconapp there is no login.html. 
please fix it

Comment (mgold1234) [reply falconapp/views.py:5]: it's a little confusing why you need falconapp directory if you have already a falcon directory?

Comment (orzionpour) [reply falconapp/views.py:5]: The problem is that I name it 'falconapp' or the fact that there are both 'falcon' and 'falconapp' directories? we were asked from barak to change what we did before and create a django application that holds the login.
All of the changes were taken as asked in the first challenge (but with name changes), including the directories names and directories tree structure.

Comment (mgold1234) [reply falconapp/views.py:5]: so maybe change it to falcon_django_apps?

**Review by aradz1997 — APPROVED** [review]

File: falconapp/views.py (hunk 1 above)
Comment (mgold1234) [reply falconapp/views.py:5]: please fix the flake8 issues, maybe put those rows with #

Comment (mgold1234) [reply falconapp/views.py:5]: ./falconapp/models.py:1:1: F401 'django.db.models' imported but unused
./falconapp/tests.py:1:1: F401 'django.test.TestCase' imported but unused
./falconapp/admin.py:1:1: F401 'django.contrib.admin' imported but unused

**Comment by orzionpour** [comment]
Closing this PR.
Will be uploaded again soon with relevant changes to the new project structure.

**Pull request closed without merging**